src/gov-action-contracts/governance/SetProposalThresholdsAndConstitutionHashAction.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../address-registries/L2AddressRegistry.sol";
+
+/// @notice Governance action for AIP 1.2 https://forum.arbitrum.foundation/t/proposal-aip-1-2-foundation-and-dao-governance/13362/22
+contract SetProposalThresholdsAndConstitutionHashAction {
```

Comment:
Should this just be named AIP1point2Action? We should also think about how we're gonna archive these upgrades so that the changes can be easily tracked